kaggle-classification/keras_trainer/model.py

```diff
@@ -171,15 +173,29 @@ def _setup_embeddings_matrix(self):
       '--job-dir', type=str, default='local_data/', help='Path to model file.')
   parser.add_argument(
       '--log_path', type=str, default='local_data/logs/', help='Path to write tensorboard logs.')
+
+  # Hyper-parameters
   parser.add_argument(
-      '--hparams', type=str, default='', help='Comma separated list of "name=value" pairs.')
+      '--learning_rate', type=float, default=0.00005, help='Learning rate.')
+  parser.add_argument(
+      '--dropout_rate', type=float, default=0.5, help='Dropout rate.')
+  parser.add_argument(
+      '--batch_size', type=int, default=64, help='Batch size.')
 
   FLAGS = parser.parse_args()
 
   hparams = DEFAULT_HPARAMS
-  hparams.parse(FLAGS.hparams)
-
-  model = ModelRunner(job_dir=FLAGS.job_dir, embeddings_path=FLAGS.embeddings_path, log_path=FLAGS.log_path, hparams=hparams)
+  hparams.learning_rate = FLAGS.learning_rate
+  hparams.dropout_rate = FLAGS.dropout_rate
+  hparams.batch_size = FLAGS.batch_size
+
+  trial_log_path = os.path.join(
+      FLAGS.log_path,
+      json.loads(
+          os.environ.get('TF_CONFIG', '{}')
+      ).get('task', {}).get('trial', '')
+  )
```

Comment:
I don't think this works. 

```
>> export TF_CONFIG=test
>> echo $TF_CONFIG
test
```
```python
import os
import json

json.loads(os.environ.get('TF_CONFIG', '{}'))

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/decoder.py", line 384, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
```

Also, I'm just curious why `TF_CONFIG`, `tastk` and `trial` are env variables while all our other parameters are flags.